research/src/sources/semantic.json5

```diff
@@ -1,698 +1,720 @@
 {
-  "$schema": "../schemas/design-system.schema.json5",
-  "lastUpdated": "2019-08-22",
-  "name": "Semantic UI",
-  "description": "Semantic is a development framework that helps create beautiful, responsive layouts using human-friendly HTML.",
-  "url": "https://semantic-ui.com",
-  "by": "Semantic Org",
-  "components": [
+  $schema: '../schemas/design-system.schema.json5',
```

Comment:
Same with this file here.

Reply:
Ah completely missed this thanks, I reverted and reapplied